src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Tasks/GetDotNetHost.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.BuildTools
+{
+    public class GetDotNetHost : Task
+    {
+        [Output]
+        public string MuxerPath { get; set; }
+
+        public override bool Execute()
+        {
+            MuxerPath = DotNetMuxer.MuxerPathOrDefault();
```

Comment:
Does the MSBuild infrastructure do this out of the box for us and shove it into a property?